editor/modes/visual-editor/invalid-block-warning.js

```diff
@@ -54,6 +62,12 @@ export default connect(
 	null,
 	( dispatch, ownProps ) => {
 		return {
+			ignoreInvalid() {
+				const { block } = ownProps;
+				const serializedBlock = serialize( { ...block, isValid: true } );
+				const parsedBlocks = parse( serializedBlock );
+				dispatch( replaceBlock( block.uid, parsedBlocks ) );
```

Comment:
Actually, we can't mutate this directly, both because it wouldn't trigger the necessary re-render, because it violates [read-only nature of state](http://redux.js.org/docs/introduction/ThreePrinciples.html#state-is-read-only), and because it wouldn't be detected by selectors which memoize by reference ([example](https://github.com/WordPress/gutenberg/blob/c8a35d4c9be0db263facb8ad9bc1f2220ed1c673/editor/selectors.js)).